lib/hangman2.rb

```diff
@@ -1,39 +1,68 @@
 class Word
   def initialize(word)
     @word = word
+    @word_as_letter = word.split('')
     @user_guesses = []
+    @lives = 6
   end
 
-class Life
-  def amount_of_lives(life)
-    @life = life
-    @amount_of_lives = 6
+
+  def amount_of_lives
+    @lives
   end
 
+  # def final_result
+  #     if @lives >= 1
+  #     letter.join == @word
+  #       #letters.join
+  #     p "bravo! you won"
+  #   end
+  # end
+
```

Comment:
A tidy and clean PR makes people happier :)
Try not to push commented-out code, if you can!